rpcs3/Emu/Cell/lv2/sys_usbd.cpp

```diff
@@ -303,6 +304,9 @@ usb_handler_thread::usb_handler_thread()
 		usb_devices.push_back(std::make_shared<usb_device_usio>(get_new_location()));
 	}
 
+	sys_usbd.notice("Adding Emulated Midi Pro Adapter");
+	usb_devices.push_back(std::make_shared<usb_device_rb3_midi_keyboard>(get_new_location()));
```

Comment:
Shouldn't this only be added if the user decides to emulate it?

Reply:
Yes. This was here for testing and the plan is to configure the devices correctly once the setting is implemented.